### PR TITLE
SCP-3794 (funny salsa hammer)

### PR DESCRIPTION
**Pull request by 77seven77seven**
## About the Pull Request

finally, a good fucking torture device.

## Why It's Good For The Game

a good torture device
good thing for stealing i guess
funny thing
we need more scps... :mistake:

## Changelog

:cl:
add: Adds SCP-3794 (funny hammer!!!)
/:cl:

### CODE REVIEW

**Comment by 77seven77seven** [comment]
guys pls merg

**Comment by checkraisefold** [comment]
this is dumb

**Comment by 77seven77seven** [comment]
> this is dumb

nuh uh

File: code/modules/SCP/SCPs/SCP-3794.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	var/obj/item/organ/external/L = H.get_organ(G.zone_sel.selecting)
+	if(ishuman(user && target))
+		if(target_zone == BP_CHEST) //that would be just too easy wouldnt it? (proceeds to not rule out the head)
+			to_chat(G, SPAN_WARNING("You can't do that!"))
+			return
+		if(L)
+			G.visible_message(SPAN_DANGER("[G] begins to swing [src] at [H]!"))
+			to_chat(H, SPAN_USERDANGER("Someone begins swinging a sledgehammer at you!"))
+			if(do_after(G, 1 SECOND, H))
+				L.droplimb(TRUE, TRUE, FALSE, TRUE)
```
Comment (checkraisefold) [inline code/modules/SCP/SCPs/SCP-3794.dm]: the second argument is supposed to be DROPLIMB_EDGE, DROPLIMB_BURN, or DROPLIMB_BLUNT

File: code/modules/SCP/SCPs/SCP-3794.dm [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	var/mob/living/carbon/human/G = user
+	var/obj/item/organ/external/L = H.get_organ(G.zone_sel.selecting)
+	if(ishuman(user && target))
+		if(target_zone == BP_CHEST) //that would be just too easy wouldnt it? (proceeds to not rule out the head)
+			to_chat(G, SPAN_WARNING("You can't do that!"))
+			return
+		if(L)
+			G.visible_message(SPAN_DANGER("[G] begins to swing [src] at [H]!"))
+			to_chat(H, SPAN_USERDANGER("Someone begins swinging a sledgehammer at you!"))
+			if(do_after(G, 1 SECOND, H))
```
Comment (checkraisefold) [inline code/modules/SCP/SCPs/SCP-3794.dm]: this seems terrible for balance

Comment (77seven77seven) [reply code/modules/SCP/SCPs/SCP-3794.dm]: dont worry i got this my man

File: code/modules/SCP/SCPs/SCP-3794.dm (hunk 1 above)
Comment (77seven77seven) [reply code/modules/SCP/SCPs/SCP-3794.dm]: fixed my man

File: code/modules/SCP/SCPs/SCP-3794.dm (hunk 2 above)
Comment (77seven77seven) [reply code/modules/SCP/SCPs/SCP-3794.dm]: fixed my man

**Comment by 77seven77seven** [comment]
fix'ed now megrge now

**Comment by 77seven77seven** [comment]
MERGE.............. NOW!!!!

**Pull request closed without merging**